examples/SimpleRagas/SimpleRagas.ipynb

```diff
@@ -0,0 +1,615 @@
+    "    \"What impact does starring a document have on other workspace members in Coda?\"\n",
+    ")\n",
+    "\n",
+    "relevant_sections = await fetch_top_k_relevant_sections(question)\n",
+    "\n",
+    "relevant_sections"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "And then passing the relevant documents to the LLM:\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Starring a document in Coda only affects the individual who starred it. It does not impact other workspace members, as the star only saves the document to the individual's personal My Shortcuts section. Other workspace members will not see the document as starred in their own views.\n"
+     ]
+    }
+   ],
+   "source": [
+    "answer = await generate_answer_from_docs(question, relevant_sections)\n",
+    "\n",
+    "print(answer)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We'll use a quick convenience function to combine these two steps, and return both the final answer and the retrieved documents so we can observe if we picked useful documents! (Later, returning documents will come in useful for evaluations)\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Starring a document in Coda only affects the individual who starred it. It does not impact other workspace members, as the star only saves the document to the individual's personal My Shortcuts section. Other workspace members will not see the document as starred in their own views.\n"
+     ]
+    }
+   ],
+   "source": [
+    "from braintrust import EvalHooks\n",
+    "\n",
+    "\n",
+    "@braintrust.traced\n",
+    "async def generate_answer_e2e(question: str):\n",
+    "    retrieved_content = await fetch_top_k_relevant_sections(question)\n",
+    "    answer = await generate_answer_from_docs(question, retrieved_content)\n",
+    "    return dict(answer=answer, retrieved_docs=retrieved_content)\n",
+    "\n",
+    "\n",
+    "e2e_answer = await generate_answer_e2e(question)\n",
+    "print(e2e_answer.get(\"answer\"))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Perfect! Now that we have the whole system working, we can compute RAGAS metrics and try a couple improvements\n",
+    "\n",
+    "## Baseline ragas metrics with autoeval\n",
+    "\n",
+    "To get a large enough sample size for evaluations, we're going to use the synthetic test questions we generated in [ our earlier cookbook ](https://www.braintrust.dev/docs/cookbook/CodaHelpDesk). Feel free to check out that cookbook if you're interested in synthetic data generation, otherwise, you can just load them from this file location:\n"
```

Comment:
the spacing in the brackets in `[ our earlier cookbook ]` results in a space before the period.